src/quo2/components/messages/new_messages_header.cljs

```diff
@@ -0,0 +1,74 @@
+(ns quo2.components.messages.new-messages-header
```

Comment:
any particular reason you called it `new-messages-header` and not just `new-messages`?
There was a decision to keep the mapping 1-1 with the figma design files so imo we should keep things consistent with that approach unless there's any strong reason not to do so

Reply:
yes i asked to rename, because i was confused by ns, it should be `dividers/new-messages`

Reply:
oh yeah this is in the wrong folder too, right? it should be in `src/quo2/components/dividers`

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components

Reply:
and if this is the case then can you adjust in the `screens` folder and the quo2 ui too 👍

Reply:
@J-Son89 -  refactoring is done, moved the component to dividers. Apologies, I didn't know about the 1:1 mapping earlier. Please check.

Reply:
cool, actually I think we made that decision after you started. All good :)